backend/handleError.ts

```diff
@@ -16,6 +16,14 @@ export function handleError(res: NextApiResponse, e: unknown) {
         typeof e.meta?.modelName === 'string' ? e.meta?.modelName : 'Record';
       return res.status(404).send(`${modelName} was not found on the server!`);
     }
+    if (e.code === 'P2002') {
+      if (Array.isArray(e.meta?.target) === true) {
+        const targetName =
+          typeof e.meta?.target[0] === 'string' ? e.meta?.target[0] : 'Record';
+        return res.status(400).send(`${targetName} was not unique!`);
+      }
+      return res.status(400).send('Record was not unique!');
```

Comment:
Tämän voisi yksinkertaistaa käyttämään samaa logiikkaa kuin P2025:
```suggestion
      const targetName =
        Array.isArray(e.meta?.target) && typeof e.meta.target[0] === 'string'
          ? e.meta.target[0]
          : 'Record';
      return res.status(400).send(`${targetName} was not unique!`);
```